packages/adminrouter/extra/src/test-harness/tests/conftest.py

```diff
@@ -33,7 +33,7 @@ def repo_is_ee():
 
 
 @pytest.fixture(scope='session')
-def mocker_s(repo_is_ee, syslog_mock, extra_lo_ips):
+def mocker_s(repo_is_ee, syslog_mock, extra_lo_ips, dns_server_mock_s):
```

Comment:
I simply meant - would be nice to make clear that this isn't used by using `usefixture`.